robot/Cumulus/resources/GiftEntryPageObject.py

```diff
@@ -580,6 +580,39 @@ def clear_lookup_value(self,field):
         self.selenium.scroll_element_into_view(locator)
         self.selenium.click_button(locator)
 
+
+    def click_table_row_button(self,title):
```

Comment:
there is already a keyword that performs this action "Perform Action On Datatable Row" please use that

Reply:
ok, thanks